Update content-style-guide.md (#4341)
fishtown-analytics/docs.getdbt.com: contributing/content-style-guide.md

## Tables
Tables provide a great way to present complex information and can help the content be more scannable for users, too.

> There are many ways to construct a table, like row spanning and cell splitting. Make sure the content is clear, concise, and presented well on the web page (like avoiding awkward word wrapping).

Reviewer: We should avoid using "like" to present examples.
```suggestion
There are many ways to construct a table, such as row spanning and cell splitting. The content should be clear, concise, and presented well on the web page (for example, avoid awkward word wrapping).
```

Author: i hear you @matthewshaver  might be worth discussing with the docs team bc the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/2db0204a02fdf9150732662b7369ba50c7a6f65f/contributing/content-style-guide.md#latin-abbreviations) itself suggests using 'like' (and i was also told to use 'like', 'such as', 'for example', as opposed to 'eg' when i started).

![Screenshot 2023-10-27 at 11 48 19](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/71288181-6d6b-4cf8-a366-c3cd8e459d81)

There won't be many instances where you need to display 4 or 5 cards on the docs site. While we recommend you use 2 or 3-column grids, you can use 4 or 5-column grids in the following scenarios:

> - For cards that contain little text and are limited to under 15 words. (This is to make sure the text isn't squished)

Reviewer: ```suggestion
- For cards that contain little text and are limited to 15 words or less. This is to make sure the text isn't squished.
```

Product names, trademarks, services, and tools should be written as proper nouns, unless otherwise specified by the company or trademark owner.

> As of October 2023, avoid using "dbt CLI" or "CLI" terminology. dbt officially provides two command line tools for running dbt commands:

Author: ```suggestion
As of October 2023, avoid using "dbt CLI" or "CLI" terminology when referring to the dbt Cloud CLI or dbt Core.  However, if referring to the command line as a tool, CLI is acceptable.

dbt officially provides two command line tools for running dbt commands:
```

- [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) &mdash; This tool allows you to develop locally and execute dbt commands against your dbt Cloud development environment from your local command line.

> - [dbt Core](https://github.com/dbt-labs/dbt-core) &mdash; This open-source tool is designed for local installation, enabling you to use dbt Core on the command line and communicate with databases through adapters.

Reviewer: Either this should say dbt CLI, or we need to go through our docs and change every instance (and page) with `dbt CLI` (of which there are a bunch) to `dbt Core`. Do you know what the preference is from a branding perspective? I thought the Core CLI was the "dbt CLI"

Reviewer 2: Done by Mirna and Matt!
